thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/views/heatmap/HeatMapViewHandler.java

```diff
@@ -107,7 +128,30 @@ public HeatMapViewResponse process(HeatMapViewRequest request) throws Exception
           heatMapBuilder = new HeatMap.Builder(dimension);
           data.put(dataKey, heatMapBuilder);
         }
-        heatMapBuilder.addCell(dimensionValue, metric.getBaselineValue(), metric.getCurrentValue());
+        if (collectionConfig != null && collectionConfig.getCellSizeExpression() != null
+            && collectionConfig.getCellSizeExpression().get(metricName) != null) {
+          String metricExpression = metricExpressions.get(metricName);
+          String numerator = metricExpression.split("/")[0];
```

Comment:
reuse result of `metricExpression.split("/")` instead of calling it twice.

Also extract "/" to a constant?